docs/products/clickhouse/howto/enable-tiered-storage.rst

```diff
@@ -0,0 +1,95 @@
+Enable tiered storage in Aiven for ClickHouse®
+==============================================
+
+.. important::
+
+   Aiven for ClickHouse® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+Learn how to enable the tiered storage feature on your Aiven for ClickHouse® service and activate it for specific tables.
+To check what the tiered storage is, how it works, and why use it, see :doc:`Tiered storage in Aiven for ClickHouse® </docs/products/clickhouse/concepts/clickhouse-tiered-storage>`.
+
+About enabling tiered storage
+-----------------------------
+
+You can enable the tiered storage feature in Aiven for ClickHouse either for a new service or for an existing one.
+To enable the feature, you need to activate it on a service level in Aiven Console and, next, set it up on a table level, which is done using SQL (via CLI, for example).
```

Comment:
Not true any more. It's always going to be on for all CH services, newly created and existing. Activation is only done via SQL. Until users activate it, they won't incur any costs, so we figured we'd just turn it on for everyone, to simplify the Console experience. There's nothing user configurable for TS for CH in Console. This was quite a recent change.